apps/studio/src/lib/db/types.ts

```diff
@@ -1,32 +1,8 @@
-import type { AlterPartitionsSpec, AlterTableSpec, IndexAlterations, RelationAlterations } from "@shared/lib/dialects/models";
-import type { CancelableQuery, DatabaseFilterOptions, ExtendedTableColumn, FilterOptions, OrderBy, PrimaryKeyColumn, Routine, SchemaFilterOptions, StreamResults, SupportedFeatures, TableChanges, TableColumn, TableFilter, TableIndex, TableInsert, TableOrView, TablePartition, TableProperties, TableResult, TableTrigger, TableUpdateResult } from "./models";
 import type { SSHConnection } from '@/vendor/node-ssh-forward/index';
-import type { RedshiftOptions, BigQueryOptions } from '@/common/appdb/models/saved_connection';
-import type { default as mysql } from './clients/mysql'
-import type { default as postgresql } from './clients/postgresql'
-import type { default as cockroach } from './clients/cockroach'
-import type { default as redshift } from './clients/redshift'
-import type { default as sqlserver } from './clients/sqlserver'
-import type { default as sqlite } from './clients/sqlite'
-import type { default as cassandra } from './clients/cassandra'
-import type { default as mariadb } from './clients/mariadb'
-import type { default as bigquery } from './clients/bigquery'
-import type { default as firebird } from "./clients/firebird";
-import type { default as oracle } from "./clients/oracle"
+import type { RedshiftOptions, BigQueryOptions, CassandraOptions } from '@/common/appdb/models/saved_connection';
```

Comment:
We can't use `models` inside of `lib`. We should add these interfaces to `shared` if we need them in both places

Reply:
(like alongside the dialect interfaces)